From 75fa7c080336f51f34751a0c46f7371b711d2a48 Mon Sep 17 00:00:00 2001
From: RickyWu-wiwynn <ricky_cx_wu@wiwynn.com>
Date: Fri, 1 Nov 2024 14:06:29 +0800
Subject: [PATCH 1007/1016] arm: dts: aspeed: yosemite4: add EMC1403 for
 Terminus NICs.

Add EMC1403 for the temperature sensor on Terminus NICs.
- Haven't contributed because there is no binding document of emc1403 on upstream.
---
 .../aspeed/aspeed-bmc-facebook-yosemite4.dts  | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index 380a0a998077..ac64a1a2594e 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -1544,6 +1544,11 @@ temperature-sensor@1f {
 				reg = <0x1f>;
 			};
 
+			emc1403@3c {
+				compatible = "smsc,emc1403";
+				reg = <0x3c>;
+			};
+
 			eeprom@50 {
 				compatible = "atmel,24c64";
 				reg = <0x50>;
@@ -1560,6 +1565,11 @@ temperature-sensor@1f {
 				reg = <0x1f>;
 			};
 
+			emc1403@3c {
+				compatible = "smsc,emc1403";
+				reg = <0x3c>;
+			};
+
 			eeprom@50 {
 				compatible = "atmel,24c64";
 				reg = <0x50>;
@@ -1576,6 +1586,11 @@ temperature-sensor@1f {
 				reg = <0x1f>;
 			};
 
+			emc1403@3c {
+				compatible = "smsc,emc1403";
+				reg = <0x3c>;
+			};
+
 			eeprom@50 {
 				compatible = "atmel,24c64";
 				reg = <0x50>;
@@ -1592,6 +1607,11 @@ temperature-sensor@1f {
 				reg = <0x1f>;
 			};
 
+			emc1403@3c {
+				compatible = "smsc,emc1403";
+				reg = <0x3c>;
+			};
+
 			eeprom@50 {
 				compatible = "atmel,24c64";
 				reg = <0x50>;
-- 
2.47.3

